src/modules/colorPicker/ColorPickerUI/Behaviors/ResizeBehavior.cs

```diff
@@ -32,12 +29,9 @@ private static void WidthPropertyChanged(DependencyObject d, DependencyPropertyC
         private static void HeightPropertyChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
         {
             var sender = ((ResizeBehavior)d).AssociatedObject;
-            var move = new DoubleAnimation(sender.Height, (double)e.NewValue, new Duration(TimeSpan.FromMilliseconds(150)), FillBehavior.Stop);
-            move.Completed += (s, e1) =>
-            {
-                sender.BeginAnimation(FrameworkElement.HeightProperty, null);
-                sender.Height = (double)e.NewValue;
-            };
+            sender.Height = (double)e.NewValue;
+
+            var move = new DoubleAnimation(sender.Height, (double)e.NewValue, new Duration(TimeSpan.FromMilliseconds(100)), FillBehavior.Stop);
```

Comment:
Stupid q, i read line 32-34 as you're setting the height to the new value then doing an animation to the same value?  32 the value is set so 34 the from and to are the same value.